gcp/modules/dcos-tested-gcp-oses/input_test_profile

```diff
@@ -1,20 +1,20 @@
 # Test Variables to Ensure Working Module
 # Run with `terraform apply -var-file test_input_profile`
 #
-# Desired Output : 
-# 
+# Desired Output :
+#
 # Apply complete! Resources: 0 added, 0 changed, 0 destroyed.
 #
 # Outputs:
 #
-# azure_offer = RHEL
-# azure_publisher = RedHat
-# azure_sku = 7.3
-# azure_version = 7.3.2017053118
+# gcp_offer = RHEL
+# gcp_publisher = RedHat
+# gcp_sku = 7.3
+# gcp_version = 7.3.2017053118
 # ------------------------------
-provider = "azure"
+provider = "gcp"
 os = "rhel_7.3"
```

Comment:
Wait, what is this? How can we select the OS we want? It's very important to the Kubernetes team to run CentOS 7.4, CoreOS (stable) and Ubuntu LTS in different, parallel test runs.

Reply:
yup